go-controller/pkg/cni/cni.go

```diff
@@ -74,23 +74,22 @@ func (pr *PodRequest) cmdAdd() ([]byte, error) {
 	// Exponential back off ~32 seconds + 7* t(api call)
 	var annotationBackoff = wait.Backoff{Duration: 1 * time.Second, Steps: 7, Factor: 1.5, Jitter: 0.1}
 	var annotations map[string]string
-	var ovnAnnotation string
 	if err = wait.ExponentialBackoff(annotationBackoff, func() (bool, error) {
 		annotations, err = kubecli.GetAnnotationsOnPod(namespace, podName)
 		if err != nil {
 			// TODO: check if err is non recoverable
 			logrus.Warningf("error getting pod annotations: %v", err)
 			return false, nil
 		}
-		if ovnAnnotation = annotations["ovn"]; ovnAnnotation != "" {
+		if _, ok := annotations[util.OvnPodAnnotationName]; ok {
```

Comment:
@girishmg don't we need a check for the legacy annotation here too?

Reply:
@dcbw will add that check.

Can you tell me when that check will be exercised?  Is it when a `kubelet` restarts on the node and it walks through the existing set of pods running on that node and call CNI CMD_ADD for each one of them?

Reply:
@girishmg Old master + new node in case both happen to be running for a short period of time. In the OpenShift case I believe the control plane (eg master) gets upgraded first. But it's two LoC so might as well cover the case here too, since everywhere else should handle it.